lib/simplecov.rb

```diff
@@ -79,9 +79,9 @@ def result
       # first, then merge the results and return those
       if use_merging
         SimpleCov::ResultMerger.store_result(@result) if @result
-        return SimpleCov::ResultMerger.merged_result
-      else
-        return @result if defined? @result
+        SimpleCov::ResultMerger.merged_result
+      elsif defined? @result
+        @result if defined? @result
```

Comment:
Whoops, I think the right move is to get rid of the now-redundant `if defined? @result`. On the way.